frontend/src/components/dashboard/CardMenuBttn.js

```diff
@@ -1,56 +1,150 @@
 
+  // Menu Functions
   const handleClick = (event) => {
     setAnchorEl(event.currentTarget);
   };
-
   const handleClose = () => {
     setAnchorEl(null);
   };
-
   const handleDelete = () => {
     dispatch(deleteArtwork(id, auth.uid));
   };
 
+  // PopUp functions
+  const handleChange = (event) => {
+    setAge((event.target.value) || '');
+  };
+  const handlePopUp = () => {
+    setOpen(true);
+    setAnchorEl(null);
+  };
+  const handlePopUpClose = () => {
+    setOpen(false);
+  };
+
   return (
-    <div>
-      <IconButton
-        aria-label="image options menu"
-        aria-controls="simple-menu"
-        aria-haspopup="true"
-        onClick={handleClick}
-      >
-        <MoreHorizIcon />
-      </IconButton>
-      <Menu
-        id="simple-menu"
-        anchorEl={anchorEl}
-        keepMounted
-        open={Boolean(anchorEl)}
-        onClose={handleClose}
-      >
-        <MenuItem onClick={handleClose}>Modify</MenuItem>
-        <MenuItem onClick={handleDelete}>Delete</MenuItem>
-      </Menu>
-    </div>
+    <>
+      <div>
+        <IconButton
+          aria-label="image options menu"
+          aria-controls="simple-menu"
+          aria-haspopup="true"
+          onClick={handleClick}
+        >
+          <MoreHorizIcon />
+        </IconButton>
+        <Menu
+          id="simple-menu"
+          anchorEl={anchorEl}
+          keepMounted
+          open={Boolean(anchorEl)}
+          onClose={handleClose}
+        >
+          <MenuItem onClick={handlePopUp}>Modify</MenuItem>
+          <MenuItem onClick={handleDelete}>Delete</MenuItem>
+        </Menu>
+      </div>
+
+      <div>
+        <Dialog
+          disableBackdropClick
+          disableEscapeKeyDown
+          open={open}
+          onClose={handleClose}
+        >
+          <form
+            name="image-uploads"
+            action={actionUrl}
+            method="POST"
+          >
+            <DialogTitle>Edit art information</DialogTitle>
+            <DialogContent>
+              <div className={classes.container}>
+                <FormControl required className={classes.formControl}>
+                  <InputLabel id="demo-dialog-select-label">Choice</InputLabel>
+                  <Select
+                    labelId="demo-dialog-select-label"
+                    id="demo-dialog-select"
+                    value={age}
+                    onChange={handleChange}
+                    input={<Input />}
+                  >
+                    <MenuItem value={'description'}>Description</MenuItem>
+                    <MenuItem value={'altText'}>Alt Text</MenuItem>
+                  </Select>
+                  <TextField
+                    required
+                    id="image-info"
+                    className={classes.padding}
+                    label="New Content"
+                    multiline
+                    name="description"
+                    placeholder="In this image..."
```

Comment:
Is it possible to populate the textfield with the previous description to make it easier to edit a previous description? Not sure how easy it is to get the description of one specific artwork so definitely not necessary if it's too much of a headache but just a thought

Reply:
I will look into this after I handle the backend. I was thinking the same thing, it would be very convenient for the user! Also I have tested with a screen reader and it works! I was worried the reader wouldn't "see" the pop up so I checked before I made the PR

Reply:
Awesome!! Sounds good